src/App/components/Contributing/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Anchor from 'package/components/Anchor'
+
+const Contributing = () => (
+  <Anchor url={'https://github.com/eggheadio/egghead-ui/blob/master/CONTRIBUTING.md'}>
+    See CONTRIBUTING.md
+  </Anchor>
+)
+
+export default Contributing
```

Comment:
Would like to inline this markdown later; could do that for the `Readme` screen of the app too